site/videosync.js

```diff
@@ -556,7 +556,25 @@ $.getScript("jquery.scrollintoview.js");
 			} else {
 				debug_print("Error: Could not find video to move.");
 			}
-		}
+		},
+
+    removeAllPrevious: function() {
+      var currentIndex = queue.current_index();
+
+      var children = $("#queue").children();
+      
+      // goes through the queue and adds the list of 
+      for (var i = 0; i < children.length; i++) {
+        var videoId = children[i].attr("data-item_id");
+
+        if (videoId === currentIndex)

```

Comment:
I recommend using brackets even for single-line statements.  Omitting them may be a source of bugs, like the famous apple security bug, where a merge issue duplicated a line and added a "goto fail;" line right in the middle of some SSL verification checks:

![iPhone SSL security bug](http://i.imgur.com/zc3lPu4.png)

This code was written before I started doing that, so I may violate this rule elsewhere in the file.